test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java

```diff
@@ -93,6 +114,75 @@ public void testSingleReplicateFile()
         assertElementNotPresent("Shouldn't have reproducibility report link", reproducibilityReportLink());
     }
 
+    @Test
+    public void testCalibrationCurve()
+    {
+        String subfolderName = "Calibration Curve";
+        goToProjectHome();
+        setupSubfolder(getProjectName(), subfolderName, FolderType.Library);
+
+        navigateToFolder(getProjectName(), subfolderName);
+        importData(SKY_FILE_FOR_CALIBRATION_CURVE);
+
+        clickTab("Proteins");
+
+        checker().verifyFalse("Reproducibility report link should not be present", isElementPresent(reproducibilityReportLink()));
+        checker().verifyTrue("Calibration curve icon should be present", isElementPresent(calibrationCurvesLink()));
+
+        clickAndWait(calibrationCurvesLink().index(2));
+
+        log("Verifying the FOM values");
```

Comment:
If we're just clicking on the third link (and not identifying it based on the protein name) let's also verify that we're getting to the peptide we're expecting, along with the upper and lower limits

Reply:
In the Protein tab, Protein/Label column  as it listed as TG|1578-1589|z2 upon clicking reproducibility report link it navigates to page with title "Calibration Curve: VIFDANAPVAVR"

Verify for sequence "VIFDANAPVAVR". ?

Reply:
Yes, sounds good.